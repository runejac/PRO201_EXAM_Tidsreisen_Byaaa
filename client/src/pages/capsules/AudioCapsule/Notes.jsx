import React from "react";
import note from "../../../assets/images/soundcapsule/note2.svg";
import singleNote from "../../../assets/images/soundcapsule/note3.svg";
const Notes = () => {
  return (
    <div>
      <div
        style={{
          position: "absolute",
          top: 150,
          left: 20,
          color: "rgba(255, 255, 255, 0.271)",
        }}
      >
        <img src={note} alt="Some note" />
      </div>
      <div
        style={{
          position: "absolute",
          bottom: 20,
          left: 20,
          color: "rgba(255, 255, 255, 0.271)",
        }}
      >
        <img width={100} src={note} alt="Some note" />
      </div>
      <div
        style={{
          position: "absolute",
          top: 200,
          right: 20,
          color: "rgba(255, 255, 255, 0.271)",
          transform: "scaleX(-1)",
        }}
      >
        <img width={80} src={note} alt="Some note" />
      </div>
      <div
        style={{
          position: "absolute",
          top: 500,
          right: 20,
          color: "rgba(255, 255, 255, 0.271)",
          transform: "scaleX(-1)",
        }}
      >
        <img width={50} src={singleNote} alt="Some note" />
      </div>
      <div
        style={{
          position: "absolute",
          top: 400,
          left: 20,
          color: "rgba(255, 255, 255, 0.271)",
        }}
      >
        <img width={50} src={singleNote} alt="Some note" />
      </div>
    </div>
  );
};

export default Notes;
