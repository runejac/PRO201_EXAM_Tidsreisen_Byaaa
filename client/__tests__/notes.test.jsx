/**
 * @jest-environment jsdom
 */

import React from "react";
import { MemoryRouter } from "react-router-dom";
import { fireEvent, render } from "@testing-library/react";
import Notes from "../src/pages/capsules/AudioCapsule/Notes.jsx";

describe("Notes", () => {
  it("shows snapshot", async () => {
    const component = render(
      <MemoryRouter>
        <Notes />
      </MemoryRouter>
    );
    expect(component).toMatchSnapshot();
  });
});
