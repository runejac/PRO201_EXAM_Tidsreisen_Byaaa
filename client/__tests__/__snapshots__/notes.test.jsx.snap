// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes shows snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          style="position: absolute; top: 150px; left: 20px; color: rgba(255, 255, 255, 0.271);"
        >
          <img
            alt="Some note"
            src="test-file-stub"
          />
        </div>
        <div
          style="position: absolute; bottom: 20px; left: 20px; color: rgba(255, 255, 255, 0.271);"
        >
          <img
            alt="Some note"
            src="test-file-stub"
            width="100"
          />
        </div>
        <div
          style="position: absolute; top: 200px; right: 20px; color: rgba(255, 255, 255, 0.271); transform: scaleX(-1);"
        >
          <img
            alt="Some note"
            src="test-file-stub"
            width="80"
          />
        </div>
        <div
          style="position: absolute; top: 500px; right: 20px; color: rgba(255, 255, 255, 0.271); transform: scaleX(-1);"
        >
          <img
            alt="Some note"
            src="test-file-stub"
            width="50"
          />
        </div>
        <div
          style="position: absolute; top: 400px; left: 20px; color: rgba(255, 255, 255, 0.271);"
        >
          <img
            alt="Some note"
            src="test-file-stub"
            width="50"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        style="position: absolute; top: 150px; left: 20px; color: rgba(255, 255, 255, 0.271);"
      >
        <img
          alt="Some note"
          src="test-file-stub"
        />
      </div>
      <div
        style="position: absolute; bottom: 20px; left: 20px; color: rgba(255, 255, 255, 0.271);"
      >
        <img
          alt="Some note"
          src="test-file-stub"
          width="100"
        />
      </div>
      <div
        style="position: absolute; top: 200px; right: 20px; color: rgba(255, 255, 255, 0.271); transform: scaleX(-1);"
      >
        <img
          alt="Some note"
          src="test-file-stub"
          width="80"
        />
      </div>
      <div
        style="position: absolute; top: 500px; right: 20px; color: rgba(255, 255, 255, 0.271); transform: scaleX(-1);"
      >
        <img
          alt="Some note"
          src="test-file-stub"
          width="50"
        />
      </div>
      <div
        style="position: absolute; top: 400px; left: 20px; color: rgba(255, 255, 255, 0.271);"
      >
        <img
          alt="Some note"
          src="test-file-stub"
          width="50"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
